src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRPollingLogAction.java

```diff
@@ -16,13 +16,13 @@
  * @author Alina Karpovich
  */
 public class GitHubPRPollingLogAction implements Action {
-    private transient AbstractProject<?, ?> project;
+    private transient Job<?, ?> project;
```

Comment:
Augh - the number of weird things that could fall under `Item` make my head explode!